domain/schema/controller/sql/0007-model.sql

```diff
@@ -99,3 +99,19 @@ INNER JOIN user AS o ON m.owner_uuid = o.uuid
 INNER JOIN life AS l ON m.life_id = l.id
 INNER JOIN model_agent AS ma ON m.uuid = ma.model_uuid
 WHERE m.activated = TRUE;
+
+-- v_model_state exists to provide a simple view over the states that are
+-- needed to calculate a model's status.
+CREATE VIEW v_model_state AS
+SELECT
+    -- Wire up the value of migrating when model migration information is
```

Comment:
Better to add a TODO tag here.

Reply:
TODO's are where thing's go to die. We are avoiding them. I have made a Jira card to track this work and linked in the PR's etc. We don't track TODO's.

Reply:
TODO means below `FALSE AS migrating` needs to be fixed in the future.

Reply:
TODO with the JIRA card number allows readers of the code to see that it's work in progress and they can then go to the jira card to see the detail and scheduling of the work. This pattern has been used previously in Juju where TODOs had to have a launchpad bug number. It worked well. Modern IDEs also track TODOs so not having them removes that essential navigation and discoverability feature.

Reply:
ok